core/solver/lower_trs.cpp

```diff
@@ -33,6 +34,35 @@ GKO_REGISTER_OPERATION(solve, lower_trs::solve);
 }  // namespace lower_trs
 
 
+template <typename ValueType, typename IndexType>
+typename LowerTrs<ValueType, IndexType>::parameters_type
+LowerTrs<ValueType, IndexType>::parse(
+    const config::pnode& config, const config::registry& context,
+    const config::type_descriptor& td_for_child)
+{
+    auto factory = LowerTrs<ValueType, IndexType>::build();
+    // duplicate?
```

Comment:
it was to indicate the duplication from lower_trs and upper_trs.
I forgot to collect them together. I have move the impl to `common_triangular_solver_parse`
(I also change the common_solver_configure to common_solver_parse now because we use parse for that)